README.md

```diff
@@ -23,7 +23,7 @@ There are a few variants of `style_text()`:
 -   `style_file()` styles a single .R file.
 -   `style_dir()` styles all .R files in a directory.
 -   `style_pkg()` styles the source files of an R package.
--   An RStudio Addin that styles the active file .R file
+-   RStudio Addins for styling the active file, styling current package and styling the highlighted code region.
```

Comment:
the current package?

Reply:
Well, the package which has its root in the current working directory (if any). Could not find a better formulation for it. Any ideas?

Reply:
styling current package -> styling the current package

Reply:
Sure.

Reply:
?